.github/workflows/ci.yml

```diff
@@ -48,11 +48,6 @@ jobs:
         run: cargo make lint-format
       - name: Check documentation
         run: cargo make lint-docs
-      - name: Check conventional commits
```

Comment:
We can also remove the conditional checkout above. (`github.event_name != 'pull_request'`)

See #77